recipe/meta.yaml

```diff
@@ -15,8 +14,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/esmf-org/esmf/archive/ESMF_{{ ver }}/esmf_{{ ver }}_src.tar.gz
-  sha256: 3693987aba2c8ae8af67a0e222bea4099a48afe09b8d3d334106f9d7fc311485
+  url: https://github.com/esmf-org/esmf/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6c39261e55dcdf9781cdfa344417b9606f7f961889d5ec626150f992f04f146d
+  patches:
+    - 0001-remove-tests-for-no-pio.patch  # [mpi == 'nompi']
+    - 0002-remove-failing-osx-tests.patch  # [osx]
```

Comment:
Definitely worth finding out why cyclic decomposition isn't working in OSX if this is still a problem in 8.4.0